adapters/revcontent/revcontent.go

```diff
@@ -0,0 +1,124 @@
+package revcontent
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/config"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+	"net/http"
+)
+
+type RevcontentAdapter struct {
```

Comment:
Nitpick: We prefer the name `adapter`, as the object is referenced by the `adapters.Bidder` interface outside of this package. There's not need to duplicate the name of the package or export it (start with an uppercase letter).